src/bidiMapper/modules/context/BrowsingContextImpl.ts

```diff
@@ -55,6 +55,8 @@ export class BrowsingContextImpl {
    * If null, this is a top-level context.
    */
   #parentId: BrowsingContext.BrowsingContext | null = null;
+  /** Flags if the initial navigation to `about:blank` is in progress. */
+  #initialNavigation = true;
```

Comment:
If the new target is created by `window.open`, the initial navigation is emitted as well

Reply:
I see navigate calls in the window.open steps https://html.spec.whatwg.org/#window-open-steps

Reply:
btw it looks like old headless implementation does not do a navigation

Reply:
> @OrKoN  I see navigate calls in the window.open steps https://html.spec.whatwg.org/#window-open-steps

The [window open steps](https://html.spec.whatwg.org/#window-open-steps) do not emit the navjgation on empty of `about:blank` urls:
> 15.4. If urlRecord is null, then set urlRecord to a [URL record](https://url.spec.whatwg.org/#concept-url) representing [about:blank](https://html.spec.whatwg.org/#about:blank).
> 15.5. If urlRecord [matches about:blank](https://html.spec.whatwg.org/#matches-about:blank), then perform the [URL and history update steps](https://html.spec.whatwg.org/#url-and-history-update-steps) given targetNavigable's [active document](https://html.spec.whatwg.org/#nav-document) and urlRecord.
> 15.6. Otherwise, [navigate](https://html.spec.whatwg.org/#navigate) ...